pkg/controller/cassandrabackup/reconciler.go

```diff
@@ -0,0 +1,140 @@
+package cassandrabackup
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/uuid"
+	cassandraoperatorv1alpha1 "github.com/instaclustr/cassandra-operator/pkg/apis/cassandraoperator/v1alpha1"
+	"github.com/instaclustr/cassandra-operator/pkg/common/operations"
+	"github.com/instaclustr/cassandra-operator/pkg/controller/cassandradatacenter"
+	"github.com/instaclustr/cassandra-operator/pkg/sidecar"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"time"
+)
+
+const (
+	controllerName = "CassandraBackupController"
+)
+
+//"github.com/instaclustr/cassandra-operator/pkg/sidecar"
+
+// blank assignment to verify that ReconcileCassandraBackup implements reconcile.Reconciler
+var _ reconcile.Reconciler = &ReconcileCassandraBackup{}
+
+// ReconcileCassandraBackup reconciles a CassandraBackup object
+type ReconcileCassandraBackup struct {
+	// This client, initialized using mgr.Client() above, is a split client
+	// that reads objects from the cache and writes to the apiserver
+	client client.Client
+	scheme *runtime.Scheme
+	recorder record.EventRecorder
+}
+
+// Reconcile reads that state of the cluster for a CassandraBackup object and makes changes based on the state read
+// and what is in the CassandraBackup.Spec
+// TODO(user): Modify this Reconcile function to implement your Controller logic.  This example creates
+// a Pod as an example
+// Note:
+// The Controller will requeue the Request to be processed again if the returned error is non-nil or
+// Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
+func (r *ReconcileCassandraBackup) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
+	reqLogger.Info("Reconciling CassandraBackup")
+
+	// backup and shit
```

Comment:
i would omit any explicit language in Instaclustr projects

Reply:
sure ;-)